modules/optimization/doc/content/source/optimizationreporters/GeneralOptimization.md

```diff
@@ -0,0 +1,50 @@
+# GeneralOptimization
+
+!syntax description /OptimizationReporter/GeneralOptimization
+
+## Overview
+
+
+## Optimization Parameters
+
+`GeneralOptimization` is also responsible for creating parameter vector(s) for
+optimization, setting the initial condition for the optimization, and setting
+parameter bounds and constraints. Although the [Optimize.md] executioner holds a single vector
```

Comment:
There are a few additons here that are different from OptimizaitonReporter. For
example, we have `num_values` and `num_values_name`. Here we can set those
values with reporter, but that wouldn't make sense for OptRep where you know the
number of values ahead of time always because its related to the measurement data.